emba

```diff
@@ -695,9 +706,20 @@ main() {
     if [[ "${ONLY_DEP}" -gt 0 ]]; then
       EMBA="${INVOCATION_PATH}" FIRMWARE="${FIRMWARE_PATH}" docker-compose run --rm emba -c './emba -f /firmware -i "$@"' _ "${ARGUMENTS[@]}"
       D_RETURN=$?
+      EMBA="${INVOCATION_PATH}" FIRMWARE="${FIRMWARE_PATH}" docker-compose run --rm emba_quest -c './emba -f /firmware -i "$@"' _ "${ARGUMENTS[@]}"
+      if [[ $? -ne ${D_RETURN} ]]; then
+        D_RETURN=1
+      fi
     else
+      local QUEST_CONTAINER_=""
+      QUEST_CONTAINER_="$(EMBA="${INVOCATION_PATH}" FIRMWARE="${FIRMWARE_PATH}" LOG="${LOG_DIR}" docker-compose run --detach --rm emba_quest -c './emba -l /logs -f /firmware -i "$@"' _ "${ARGUMENTS[@]}")"
+      print_output "[*] Quest docker container ${QUEST_CONTAINER_} started and detached.\\n" "main"
       EMBA="${INVOCATION_PATH}" FIRMWARE="${FIRMWARE_PATH}" LOG="${LOG_DIR}" docker-compose run --rm emba -c './emba -l /logs -f /firmware -i "$@"' _ "${ARGUMENTS[@]}"
       D_RETURN=$?
+      # stop quest container
+      if [ "$( docker container inspect -f '{{.State.Status}}' "${QUEST_CONTAINER_}" )" = "running" ]; then
```

Comment:
should this be something like `== "running"`
please use [[ and ]] instead of [ and ]